src_rebuild/premake5.lua

```diff
@@ -16,9 +16,7 @@ workspace "REDRIVER2"
     configurations { "Debug", "Release" }
 
     defines { VERSION } 
-
-    filter "system:Windows"
-        defines { "USE_32_BIT_ADDR", "PGXP" }
+    defines { "USE_32_BIT_ADDR", "PGXP" }
```

Comment:
Not correct as a global defininition, it should use collection of non-PSX platforms instead